src/mlpack/core/optimizers/sgdr/sgdr.hpp

```diff
@@ -76,7 +76,7 @@ class SGDR
    * @param updatePolicy Instantiated update policy used to adjust the given
    *        parameters.
    */
- SGDR(DecomposableFunctionType& function,
+  SGDR(DecomposableFunctionType& function,
       const size_t epochRestart = 50,
```

Comment:
The parameter should align with the first parameter: ``DecomposableFunctionType& function,``.